frontend/packages/schema-model/src/lib/mappers/custom-properties.ts

```diff
@@ -11,14 +11,16 @@ const specialAttributes = [
 
 // Deals with custom properties... or really what properties that we not know about.
 export const findCustomAttributes = (schemaNode: KeyValuePairs) => {
-  const outout: KeyValuePairs = {};
+  const output: KeyValuePairs = {};
+  console.log(schemaNode);
```

Comment:
Can this console.log statement be removed?

Reply:
Whoops 😬